daq/panda/testing/daqTestScript.py

```diff
@@ -45,9 +45,13 @@
 #Account for 1600 ms delay in Arduino connection
 sleep(1.6)
 
-numValues = 22;
 n = 0
 while(n<50):
+  now = datetime.now() #set time
+  current_time = now.strftime("%H:%M:%S:%f") [:-3] #set current time
+  previousTime = timeName
+  timeName = current_time
+  
```

Comment:
Does this code exist in the actual DAQ code? If so, this code really should not be copied and pasted into here as is, because it will very quickly get out of sync with the actual script. If the actual script doesn't have a class or routine that's callable, then that's probably what you need to address first.

Reply:
I believe that addressing this is one of Kenson's tasks for this sprint, so it should be rectified by Sunday

Reply:
I was under the impression that the code was there to stay. That snippet is used to upload the data to Firebase under the right time. However, the previousTime stuff can be removed as we're no longer using the "Previous Time" parameter in the Firebase.

Reply:
Removed superfluous time variables!